languages/java/exercises/concept/case-statement/.meta/src/reference/java/PlayAnalyzer.java

```diff
@@ -0,0 +1,55 @@
+public class PlayAnalyzer{
+
+    //enhanced switch implementation
```

Comment:
It looks like this enhanced switch implementation is using language features that are not introduced in this exercise. As per [the docs](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#example-implementation-file):

> The implementation should only use language features introduced by the exercise or its prerequisites (and their prerequisites, and so on).

There are basically two options here: 

1. Use the regular switch here that is explained in the introduction
1. Also introduce the enhanced switch syntax

Which option to choose depends on what you want to do with the extended switch syntax. Could that possible be explained in a new exercise or does that not make sense?

Reply:
What are your opinion on that ?
I'm thinking of deleting the enhanced switch syntax example. Even if I really like it and I think it's one of the best improvement Java has received this last few years and it's maybe too far for a new Java developer. 
However, I'm still interested in your point of view on that.

Reply:
@ErikSchierboom  In the end, I have made the choose of the traditional switch implementation in the reference solution and explaining the enhanced one in the after.md file with the resolution of the exercise at the end.